RecoBTag/Combined/test/testDeepCSV_cfg.py

```diff
@@ -29,7 +29,7 @@
       'pfDeepCSVJetTags:probb', 
       'pfDeepCSVJetTags:probc', 
       'pfDeepCSVJetTags:probbb', 
-      'pfDeepCSVJetTags:probcc',
+##      'pfDeepCSVJetTags:probcc',
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

Reply:
@mverzett  can provide more info on this (this is for DeepCSV not DeepFlavour), but I commented out because the probcc category was merged with probc and called like the later. I will add an explanatory comment.

Reply:
added comment in https://github.com/cms-sw/cmssw/pull/19893/commits/ae625211aed5d1e232c4f1798c4c22122cfff31e